site/_layouts/doc.liquid

```diff
@@ -115,6 +115,7 @@
                     <li {% if page.url == "/docs/v1/tech/apis/lambdas/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/lambdas/">Lambdas</a></li>
                     <li {% if page.url == "/docs/v1/tech/apis/login/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/login/">Login</a></li>
                     <li {% if page.url == "/docs/v1/tech/apis/passwordless/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/passwordless/">Passwordless</a></li>
+                    <li {% if page.url == "/docs/v1/tech/apis/recovery-codes/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/recovery-codes/">Recovery Codes</a></li>
```

Comment:
Please also add site/docs/v1/tech/apis/index.adoc

Reply:
Ah, that makes total sense to me. To avoid merge issues, might make sense to do this after #578 is merged.

Reply:
Ah yes, I originally thought about this but wanted to get things moving.  My brain was in a complete fog last week so disregard any questionable decisions I might have made :)